src/python/pants/testinfra/BUILD

```diff
@@ -13,36 +12,18 @@ python_library(
     'src/python/pants/testinfra/engine:engine_test_base',
     'src/python/pants/testinfra/subsystem',
   ],
+  # TODO(???): Once we remove `pants_test:testinfra`, reclaim the wheel name
+  #  `pantsbuiuld.pants.testinfra`. We let that keep the ideal name for backwards compatibility.
   provides=pants_setup_py(
-    name='pantsbuild.pants.testinfra',
+    name='pantsbuild.pants.testinfra.new_namespace',
```

Comment:
Consider just 'pantsbuild.pants.tests' ior '''pantsbuild.pants.testsupport' - not as good names but probably good enough.

Reply:
I like `testsupport` or `testutil` (`tests` sounds like it contains actual tests).